style.css

```diff
@@ -80,8 +86,145 @@
     padding-left: 5px;
   }
 
+  .ul-items > li {
+    padding: 8px 12px;
+    list-style: none;
+    border-radius: 4px;
+    font-size: 12px;
+    font-weight: 600;
+    background: #ebf0ee;
+  }
+
   .links > ul > li {
     list-style: none;
     margin: 10px;
   }
+
+  .work {
+    padding: 64px 24px;
+  }
+
+  .title {
+    font-size: 40px;
+    font-weight: 700;
+    color: #172b4d;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    margin-bottom: 20px;
+  }
+
+  .work-container {
+    display: grid;
+    grid-template-columns: 100%;
+    grid-template-rows: repeat(6, 474px);
+    row-gap: 15px;
+  }
+
+  .work-item {
+    background-image: url("./media/plaholder.png");
+    background-repeat: no-repeat;
+    border-radius: 8px;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    border: 1px solid lightgray;
+  }
```

Comment:
Please make sure that the background fit the container when you resize the screen. :eyes: 
![image](https://user-images.githubusercontent.com/98363075/191756294-cea90e55-8fcb-4aac-8d85-4b1e294061d4.png)